lib/HAPServiceNode.js

```diff
@@ -146,6 +147,7 @@ module.exports = function(RED) {
                     manufacturer: node.manufacturer,
                     serialNo: node.serialNo,
                     model: node.model,
+                    firmwareRev: node.firmwareRev || '1.0'
```

Comment:
Please remove optional value.

Reply:
Done. I tested it with my setup and it seems to work nicely.